source/Camera/Basic/Capture2D.m

```diff
@@ -0,0 +1,52 @@
+try
+    app = zividApplication; 
+
+    disp('Connecting to camera')
+    camera = connectCamera(app);
+
+    disp('Setting the capture settings');
+    settings2D = Zivid.NET.Settings2D();
+    settings2D.Iris = 22;
+    settings2D.ExposureTime = Zivid.NET.Duration.FromMicroseconds(10000);
+    settings2D.Gain = 1.0;
+
+    disp('Capture a 2D frame');
+    frame2D = camera.Capture2D(settings2D);
+
+    disp('Get RGBA8 image from Frame2D');
+    image = NET.invokeGenericMethod(frame2D,'Image',{'Zivid.NET.RGBA8'});
+
+    disp(['Image Width: ',  num2str(image.Width), ' Height: ',  num2str(image.Height)]);
+    disp(['Pixel (0, 0): R=', num2str(image.Item(1).r), ', G=', num2str(image.Item(1).g), ', B=', num2str(image.Item(1).b), ', A=', num2str(image.Item(1).a)]);
```

Comment:
I don't see why we have this. I think the imshow at the end is enough. Also Image.Item is zerobase, so you are giving pixel (1,1) not (0,0) here.

Reply:
Regarding whether we should have it - I think we should show both how to access data (pixel values) and the whole object (image matrix). But I don't see why we should have two different ways to do the same thing, i.e. image2D.Item(0).r and byteArray(0). However, the Matlab script is a copy of the equivalent C# sample. Therefore, I think we should keep it for now but put this to question for the C# sample. Then, if we change the C# sample, we update the Matlab sample accordingly.

I've changed the indexing to:

disp(['Pixel (0, 0): R=', num2str(image2D.Item(0).r), ', G=', num2str(image2D.Item(0).g), ', B=', num2str(image2D.Item(0).b), ', A=', num2str(image2D.Item(0).a)]);

 disp(['First four bytes:  [0]: ',num2str(byteArray(0)),', [1]: ',num2str(byteArray(1)),', [2]: ',num2str(byteArray(2)),', [3]: ',num2str(byteArray(3))]);

Reply:
Decided to keep it for consistenncy for now.